framework/include/mesh/MooseMesh.h

```diff
@@ -140,6 +140,12 @@ class MooseMesh :
   std::map<dof_id_type, std::vector<dof_id_type> > & nodeToElemMap();
 
   /**
+   * If not already created, creates a map from every node to all
+   * _active_ elements to which they are created.
```

Comment:
To address @friedmud's comment. I would update this description to mention that extra ghosted elems are NOT included in your map. We can open another ticket to add that later since it'll mean a little refactoring in FEProblem anyway.
 Also, there's I think you meant to say "connected" here anyway so the existing description is already horked.